### PR TITLE
Add initial app files

### PR DESCRIPTION
**Pull request by Eugineer0**

### CODE REVIEW

File: Admin/AdminApp/karma.conf.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+module.exports = function (config) {
```
Comment (KseniyaMikhailiuk) [inline Admin/AdminApp/karma.conf.js]: have you read about karma.conf.js file? what is its purpose?

File: Admin/AdminApp/package.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "watch": "ng build --watch --configuration development",
+    "test": "ng test"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "~12.2.0",
```
Comment (KseniyaMikhailiuk) [inline Admin/AdminApp/package.json]: what ~ ^ symbols mean? which is better to use here?

File: CarRentalClientSide/ClientApp/src/app/app.module.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+
+import { AppComponent } from './app.component';
+
+@NgModule({
+  declarations: [
+    AppComponent,
```
Comment (kseniya-mikhailiuk-simpligov) [inline CarRentalClientSide/ClientApp/src/app/app.module.ts:8]: useless comma

Comment (KseniyaMikhailiuk) [inline CarRentalClientSide/ClientApp/src/app/app.module.ts]: useless comma

**Review by KseniyaMikhailiuk — APPROVED** [review]

**Review by andrei-kruhliakou-itechart — APPROVED** [review]

**Pull request merged**